data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,622 @@
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/fusion engine large"
+	"mass" 297
+	"outfit space" -99
+	"engine capacity" -99
+	"thrust" 360
+	"thrusting energy" 27
+	"thrusting heat" 135
+	"thrusting fuel" 0.27
+	"drag reduction" 0.9
+	"flare sprite" "effect/avgi flare/fusion"
+		"frame rate" 30
+		"scale" 2
+	"flare sound" "zpinch"
+	"afterburner effect" "large remass injection"
+	"magnozzle rings" 9
+	description "The largest and oldest of the common fusion torches used by the Avgi, the Arc is derived from an interstellar-class design that was modified and downrated to make it more suitable for intra-system operations, trading specific impulse for raw thrust. At this scale, the pinch is long enough that it is able to confine the plasma for sufficient time to not only fuse the deuterium into a mix of helium-3 and tritium, but to likewise burn the aforementioned fusion products together with leftover deuterium to release even more energy. This reaction, known as catalyzed D-D fusion, gives the Arc unprecedented fuel efficiency over its smaller cousins."
+	description "The magnetic nozzle is designed to support a variety of components that can be slotted in to provide various effects, from injecting additional propellant to improve thrust to tapping energy from the expanding fusion plasma to power a ship's systems."
+
+
+effect "small remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 0.75
+	sound "flamethrower"
+
+effect "medium remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+	sound "flamethrower"
+
+effect "large remass injection"
+	sprite "effect/avgi flare/afterburner"
+		"frame rate" 30
+		"scale" 2
+	sound "flamethrower"
+
+
+outfit "Remass Injector"
+	series "Avgi: Engines"
+	index 100
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -3
+	"engine capacity" -3
+	"afterburner thrust" 72
+	"afterburner energy" 3
+	"afterburner heat" -12
+	"afterburner fuel" 0.3
+	"magnozzle rings" -1
+	description "A powerful magnetohydrodynamic turbopump forces a torrent of lithium-seeded water into the throat of a fusion torch's magnetic nozzle, dramatically increasing thrust at the cost of a significant increase in propellant consumption. The additional mass flow likewise reduces the waste heat generated by a torch drive, capturing much of it as propellant is injected around the fusion reaction."
+	description "The Remass Injector is a component that can be installed into an Avgi fusion torch to give it afterburning capabilities. It requires a magnozzle ring slot to be installed."
+
+
+outfit "Inductive Extractor"
+	series "Avgi: Engines"
+	index 90
+	category "Engines"
+	licenses
+		"Twilight Guard"
+	"cost" 2760000
+	thumbnail "outfit/unknown"
+	"mass" 12
+	"outfit space" -12
+	"engine capacity" -6
+	"energy generation" 1.5
+	"heat generation" 1.5
+	"thrust" -9
+	"thrusting energy" -30
```

Comment:
```suggestion
	"thrusting energy" -16.5
```
This number is balanced based on the folllowing assumptions:

1: One point of engine space is five times more valuable than one point of outfit space. Hence, the -12 outfit space and -6 engine capacity becomes an 'effective' cost of 42 space. Accounting also for the engine space cost of the thrust penalty (also considered at 5x more valuable than outfit space), and the cooling requirements, and the outfit space to power the engine (this is all assuming Avgi engines, specifically the Z-330. Normally I wouldn't be happy making such an assumption, but in this case you cannot install the inductive extractor without using Avgi engines, so I think it's okay), this works out as a total effective adjusted outfit space cost of 67.14

2: Generating energy only by thrusting is around 66% as useful as generating the same amount of energy normally.
As you point out [here](https://github.com/endless-sky/endless-sky/pull/9172/files?file-filters%5B%5D=.txt&show-viewed-files=true#r1672958615), it is indeed less valuable, and this puts a number on that.

If you accept both these assumptions, then this value makes the Inductive Extractor roughly as good of a power source as a Double Plasma Core or a Yellow Sun. Conversely, if these assumptions are wrong then this number will also be wrong. But, if you're confident about the direction in which the assumptions are wrong, then you can work backwards from that to determine whether this value is too high or too low.